scripts/artifactory-publish.sh

```diff
@@ -5,4 +5,18 @@ echo 'Deploying navi webservice models to artifactory'
 
```

Comment:
Can we delete the `echo 'Deploying navi webservice models to artifactory'` line^